SalesforceFoundation/EDA
# [W-10187060] Release Gating Card for Spring 22

# Changes

- Spring '22 card will now appear under Release Management in Education Cloud Settings. 

# Issues Closed
[W-10187060
](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000PnDSRYA3/view)

# New Metadata

* Labels
   * stgReleaseGateEDASpring22Label
   * stgReleaseGateEDASpring22Description
   * stgReleaseGateEDASpring22FeatureDescription
   * stgReleaseGateEDASpring22FeatureHelpLabel

* Apex Class
   * EDASpring22ReleaseGate.cls
   * EDASpring22ReleaseGate_TEST.cls  


# Testing Notes

1. Go to Education Cloud Settings tab
2. Click on release management
3. Under "Education Data Architecture", it should have a card with the following details

- Card header should say "Spring '22"
- Description header should say, "There were no features gated for this release therefore all features were automatically released with Spring'22"
- Bullet item 1 header should say, "Please see EDA's release notes for Spring'22 for full details.
- Bullet item 2 header should say, "Activate Spring'22"
- EDA release gating card should direct users to https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHOcSAM?tab=discussion&sort=LAST_MODIFIED_DATE_DESC
